src/py42/sdk/queries/fileevents/v2/filters/destination.py

```diff
@@ -0,0 +1,92 @@
+from py42.choices import Choices
+from py42.sdk.queries.fileevents.util import _FileEventFilterStringField
+from py42.sdk.queries.query_filter import _QueryFilterStringField
+
+
+class Destination:
```

Comment:
I'm not _crazy_ about the nested classes. The method docs also get pretty crammed. In general I think there's a better way we could organize the filters, particularly with the V2 data model.  I didn't want to completely change how they work here but maybe something to consider with the future Incydr sdk.